src/Build/Evaluation/Conditionals/Scanner.cs

```diff
@@ -322,13 +336,17 @@ private static int ScanForPropertyExpressionEnd(string expression, int index)
                         {
                             nestLevel--;
                         }
+                        else if (character == ' ')
```

Comment:
Use `char.IsWhitespace` instead of just `' ' `

Reply:
Done